lib/index.js

```diff
@@ -35,6 +36,17 @@ function loadConfig(configPath) {
       'lint-staged.config.js',
       'lint-staged.config.cjs',
     ],
+    loaders: {
+      '.yml': (filepath, content) => {
+        return yaml.load(content)
+      },
+      '.yaml': (filepath, content) => {
+        return yaml.load(content)
+      },
+      noExt: (filepath, content) => {
+        return yaml.load(content)
+      },
```

Comment:
I would prefer no function body in these simple cases, and it could even be hoisted :

```js
const jsYamlLoad = (filepath, content) => yaml.load(content)

...

    loaders: {
       '.yml': jsYamlLoad,
       '.yaml': jsYamlLoad,
       noExt: jsYamlLoad,
```